typedoc-plugin-audius-theme/typedoc-plugin-audius-theme

```diff
@@ -0,0 +1 @@
+/Users/ray/Documents/Audius/docs.audius.org/typedoc-plugin-audius-theme
```

Comment:
I think there might be something wrong with the `init-sdk-docgen` command creating this file for some reason

Reply:
Oh, seems like it's not idempotent. Running it multiple times causes it to create this file

Reply:
fun! i can try to fix that on this PR

Reply:
Thanks! I don't know enough about symlinks to understand why this is happening haha